README.md

```diff
@@ -31,14 +31,15 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. What is the DOM?
-
+- The DOM is the Document Object Model
```

Comment:
I would like to see a more thorough answer. Think of these questions as interview prep.